index.html

```diff
@@ -164,6 +164,39 @@
     </nav>
 
     <main>
+      <div class="roles-container">
+        <div class="role-details">
+          <h2 class="role-heading">Select your role</h2>
+          <div class="role-details-field">
+            <label class="checkbox-label">
+              <input type="checkbox" name="developer" class="checkbox-input" />
+              Developer
+            </label>
+            <label class="checkbox-label">
+              <input type="checkbox" name="designer" class="checkbox-input" />
+              Designer
+            </label>
+            <label class="checkbox-label">
+              <input type="checkbox" name="maven" class="checkbox-input" />
+              Maven
+            </label>
+            <label class="checkbox-label">
+              <input
+                type="checkbox"
+                name="productmanager"
+                class="checkbox-input"
+              />
+              Product Manager
```

Comment:
+1

Reply:
Done with changes. can you please check latest commit message